### PR TITLE
dex 1012 - Add migration specific status messages

### PR DESCRIPTION
**Pull request by Emily-Ke**
- Adds migration information to the "Curation" finished message and "Identification" skipped message
migration-status

### CODE REVIEW

File: locale/en.json [hunk 1]
```diff
@@ -142,6 +142,7 @@
   "CURATION_SKIPPED_MESSAGE": "Curation skipped - no images to curate.",
   "IDENTIFICATION_SKIPPED_MESSAGE": "Identification skipped.",
   "IDENTIFICATION_SKIPPED_NO_IMAGES_MESSAGE": "Identification skipped - no images to match.",
+  "IDENTIFICATION_SKIPPED_MIGRATED_MESSAGE": "Identification skipped - migrated sighting.",
```
Comment (brmscheiner) [inline locale/en.json:145]: How about "Identification skipped - this sighting was migrated from the Zebra Wildbook" ?

**Review by brmscheiner — APPROVED** [review]

File: locale/en.json (hunk 1 above)
Comment (Emily-Ke) [reply locale/en.json:145]: Is there an instance ID that we have access to that we could use to know that this instance is zebra?

Comment (brmscheiner) [reply locale/en.json:145]: Ahh, no. How about "Identification skipped - this sighting was migrated from a Wildbook database."

File: src/constants/wildbookSystemGuids.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const wildbookSystemGuids = {
+  'e144fec3-2f47-4195-8025-a8915f9a52a1': 'Wildbook for Zebras',
```
Comment (Emily-Ke) [inline src/constants/wildbookSystemGuids.js:2]: This is the system GUID for zebra.wildme.org. It is  a site-setting that is a unique guid per instance.

**Comment by Emily-Ke** [comment]
There is a GUID that identifies a unique Codex instance, so on production instances, we should be able to have more specific migrated messages:

**Production migration (hopefully)**
migration-specific

**Non-production migration**
migration-general

**Non-migration**
default

File: src/constants/wildbookSystemGuids.js (hunk 2 above)
Comment (brmscheiner) [reply src/constants/wildbookSystemGuids.js:2]: would this be a `wildbookSystemGuid` or a `codexSystemGuid`?

**Review by brmscheiner — APPROVED** [review]

File: src/constants/wildbookSystemGuids.js (hunk 2 above)
Comment (Emily-Ke) [reply src/constants/wildbookSystemGuids.js:2]: It is a codex system GUID. What do you think of changing the name to `wildbooksBySystemGuids` then?

If we thought we were going to store other information too, I could change it too something like
```
const systemInformation = {
  'e144fec3-2f47-4195-8025-a8915f9a52a1': {
    migratedFrom: 'Wildbook for Zebras',
    // ... other information
  }
};
```

Perhaps this isn't even data that the front end should be storing, but something that we should request from houston.

Comment (Emily-Ke) [reply src/constants/wildbookSystemGuids.js]: Renamed to `wildbookBySystemGuid` in https://github.com/WildMeOrg/codex-frontend/pull/386/commits/f346ab342487247e98ac4b0799299d54cd94bbaa

**Pull request merged**